tests/test_obs_test.py

```diff
@@ -0,0 +1,50 @@
+import os
+import sys
+import unittest
+
+from lsst.utils import getPackageDir
+import lsst.afw.geom
+import lsst.obs.base.tests
+import lsst.obs.test
+
+
+class TestObsTest(lsst.obs.base.tests.ObsTests, unittest.TestCase):
+    def setUp(self):
+        product_dir = getPackageDir('obs_test')
+        self.data_dir = os.path.join(product_dir, 'data', 'input')
+        self.Mapper = lsst.obs.test.TestMapper
```

Comment:
I wonder if you will find you want the class or an instance. If the class, please consider appending Class to the instance variable name, for clarity (self.MapperClass is clearly not an instance of a Mapper)